+hw/+ptb/Window.m

```diff
@@ -612,14 +612,32 @@ function applyCalibration(obj, cal)
       Screen('LoadNormalizedGammaTable', obj.PtbHandle, gammaTable);
     end
     
-    function c = calibration(obj, dev, lightIn, clockIn, clockOut)
+    function c = calibration(obj, dev, lightIn, clockIn, clockOut, makePlot)
```

Comment:
can we follow the docstring format starting with this pr?

Reply:
Do you want to go ahead and do this?

Reply:
sure I can do it for this pr if you'll do it in the future : ) 
I know you want to merge asap, so I'll do it in a separate branch based off of this one called something like `expServerTest_docstring`, so after this is merged just leave the branch open